From 8d18f92b6c2b2b68a6c0c2213a9d06f5dc1944b5 Mon Sep 17 00:00:00 2001
From: Colin Guthrie <colin@mageia.org>
Date: Wed, 23 Jan 2013 23:51:48 +0000
Subject: [PATCH 103/103] Fix installation of zeromq.

This avoids a double DESTDIR in the install
(at least with standard Mageia %makeinstall macro)
---
 mythtv/external/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git mythtv/external/Makefile mythtv/external/Makefile
index 6fc0c2e..0555477 100644
--- mythtv/external/Makefile
+++ mythtv/external/Makefile
@@ -28,7 +28,7 @@ zeromq/Makefile:	zeromq/configure
 	             --includedir=${PREFIX}/include/mythtv/zeromq)
 
 zeromq-install zeromq-uninstall zeromq-clean zeromq-distclean:
-	${MAKE} -C zeromq ${@:zeromq-%=%} DESTDIR=${INSTALL_ROOT}
+	${MAKE} -C zeromq ${@:zeromq-%=%}
 
 nzmqt-all:	zeromq-all
 	(cd nzmqt/src ; ${QMAKE} -o Makefile)
-- 
1.8.1

